bin/dasht-query-line

```diff
@@ -87,19 +97,21 @@
 # Distributed under the terms of the ISC license (see the LICENSE file).
 
 : ${DASHT_DOCSETS_DIR:=${XDG_DATA_HOME:-$HOME/.local/share}/dasht/docsets}
+: ${DASHT_FRAGMENTS_CONF:=${XDG_DATA_HOME:-$HOME/.local/share}/dasht/fragments.conf}
```

Comment:
Configuration files should go under the `$XDG_CONFIG_HOME` prefix, which is `$HOME/.config` by default.